cicd/deploy_ephemeral_env.sh

```diff
@@ -8,7 +8,9 @@ source ${CICD_ROOT}/_common_deploy_logic.sh
 # -> use this PR's newly built image in the deployed configurations
 set -x
 export BONFIRE_NS_REQUESTER="${JOB_NAME}-${BUILD_NUMBER}"
+set -e
 export NAMESPACE=$(bonfire namespace reserve)
+set +e
```

Comment:
I think you can leave the 'set -e' for the deploy command as well

In fact I thought 'set -e' was already set here at this point but I guess you saw it was not?